src/components/Button/Button.module.css

```diff
@@ -215,22 +215,17 @@
 /**
  * Button Icon
  * 1) Set SVG icon fill to the current text color
- * 2) Rotate loading icon
  */
 .button__icon {
   fill: currentColor; /* 1 */
-
-  .button.eds-is-loading & {
-    animation: rotateIcon 2s linear infinite; /* 2 */
-  }
```

Comment:
I moved this `.eds-is-loading` stuff down because I was getting a lint error about descending specificity by having this before the `.button__icon + .buton__text` stuff:
![the lint error in VS code](https://user-images.githubusercontent.com/7761701/157952142-dffdeaa7-8dbc-47b0-ac2c-85a02ffd74cd.png)